### PR TITLE
List method for IssueController

### PR DESCRIPTION
**Pull request by Yarik2308**
https://github.com/malinink/easy-tests/issues/333

- [x] Method
- [x] Test
- [x] Add ACL
- [x] testListNotFound
- [x] testListForbidden
- [x] testListSubjectIdNotInt

### CODE REVIEW

**Comment by Yarik2308** [comment]
@malinink Wait Room for issueSupport.

**Comment by malinink** [comment]
@Yarik2308 look at PR, it seems you revert back comments.

**Comment by Yarik2308** [comment]
@malinink oops... Sorry. I just created this branch before pulling branch with model/mapper etc

**Comment by Yarik2308** [comment]
@malinink test pass in my PC. But without Support class in dev it will not pass here. I can copy it from https://github.com/malinink/easy-tests/issues/237 and before merge delete it.

**Comment by Yarik2308** [comment]
@malinink Do i have too test all exceptions in that PR?

**Comment by Yarik2308** [comment]
@malinink Do I have to test Subject not Integer?

**Comment by Yarik2308** [comment]
@malinink Wait Room for issueSupport.

**Comment by Yarik2308** [comment]
@malinink Ready for review

**Comment by Yarik2308** [comment]
@malinink Ready for review

File: src/main/java/easytests/api/v1/controllers/IssuesController.java [hunk 1]
```diff
@@ -1,33 +1,63 @@
...
+        if (subjectModel == null) {
+            throw new NotFoundException();
+        }
+
+        if (!this.acl.hasAccess(subjectModel)) {
+            throw new ForbiddenException();
+        }
+
+        final List<IssueModelInterface> issuesModels = this.issuesService
+                .findBySubject(new SubjectModelEmpty(subjectId));
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/IssuesController.java]: use subjectModel you already have.

File: src/main/java/easytests/api/v1/mappers/IssuesMapper.java [hunk 2]
```diff
@@ -9,4 +12,14 @@
  */
 @Service("IssuesMapperV1")
 public class IssuesMapper extends ModelMapper {
+    public IssuesMapper() {
+        super();
+        this.createTypeMap(IssueModel.class, Issue.class)
+            .addMappings(
+                mapper -> mapper.when(
```
Comment (malinink) [inline src/main/java/easytests/api/v1/mappers/IssuesMapper.java]: no need of when

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by Yarik2308** [comment]
@malinink Done.

File: src/main/java/easytests/api/v1/controllers/IssuesController.java [hunk 3]
```diff
@@ -1,33 +1,61 @@
 package easytests.api.v1.controllers;
 
+import easytests.api.v1.exceptions.ForbiddenException;
 import easytests.api.v1.mappers.IssuesMapper;
+import easytests.api.v1.models.Issue;
+import easytests.common.exceptions.NotFoundException;
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/IssuesController.java]: use not found from v1 package

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by Yarik2308** [comment]
@malinink Done (x2).

**Review by malinink — APPROVED** [review]

**Pull request merged**